tests/problems.py

```diff
@@ -156,7 +156,7 @@ def f(self, t, y):
 
     def g(self, t, y):
         ty = torch.cat([t.expand(y.size(0), 1), y], dim=1)
-        return self.g_net(ty)
+        return 0.1 * self.g_net(ty)  # small noise makes passing adjoint tests easier/possible
```

Comment:
To what extent does this matter?

Reply:
Sorry, looks like I missed this one. I found that this was needed to make passing the adjoint-against-sdeint test reliable for some of the Ito solvers. This is a test new in this PR; we hadn't previously been testing it.